### PR TITLE
WEB-3674 Improvements to prism code highlighting of Python.

### PR DESCRIPTION
**Pull request by basta**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docs/static/js/prism-python-plugin.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        lookbehind: true,
+        alias: 'property',
+        inside: {
+            'punctuation': /<|>/
+        }}
+    })
+
+    Prism.languages.python = Prism.languages.insertBefore("python", "comment", {
+        libraries: {
+            pattern: /\b(?:os|time|numpy|Path)\b/,
```
Comment (Mara3l) [inline docs/static/js/prism-python-plugin.js:20]: Add also `sdk.` and `GoodDataSDK.`

**Review by Mara3l — CHANGES_REQUESTED** [review]

File: docs/static/js/prism-python-plugin.js (hunk 1 above)
Comment (Mara3l) [reply docs/static/js/prism-python-plugin.js:20]: Also, please make it highlight Java(Type)Script and bash as well. Thanks!

**Comment by hkad98** [comment]
There are a lot of commits, please, squash them into one commit.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by Mara3l — APPROVED** [review]

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR236

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**